src/templates/account/login.html

```diff
@@ -1,23 +1,20 @@
 {% extends "base_no_sidebar.html" %}
 {% load crispy_forms_tags %}
 {% load i18n %}
-{% load account %}
+{% load account socialaccount %}
 
 {% block head_title %}{% trans "Sign In" %} | {% endblock %}
 
 {% block heading %}{% trans "Sign In" %}<i class="pull-right fa fa-sign-in"></i>{% endblock %}
 
 {% block content %}
-  {% if socialaccount.providers %}
-    <p>{% blocktrans with site.name as site_name %}Please sign in with one
-    of your existing third party accounts. Or, <a href="{{ signup_url }}">sign up</a>
+  {% if config.enable_google_signin %}
```

Comment:
I think these instruction could flow more logical, something like this?  Also, probably shouldn't say "Google" before the list that could in the future have other providers
````
  <p>Sign in via:</p>

    <div class="socialaccount_ballot">

      <ul class="socialaccount_providers">
        {% include "socialaccount/snippets/provider_list.html" with process="login" %}
      </ul>
      <a href="{% provider_login_url 'google' process='login' %}" class="btn btn-success"><i class="fa fa-google"></i>oogle</a>

      <div class="login-or">{% trans 'or' %}</div>

    </div>
  {% include "socialaccount/snippets/login_extra.html" %}
  {% endif %}

    <p>{% blocktrans %}Sign in with your username and password. If you have not created an account yet, then please
    <a href="{{ signup_url }}">sign up</a> first.{% endblocktrans %}</p>
  {% endif %}
  <form class="login" method="POST" action="{% url 'account_login' %}">
````

Reply:
Does this look right?

<img width="1059" alt="image" src="https://user-images.githubusercontent.com/10972027/230768628-275bfa8e-cf48-42b5-98db-39f4186bafc6.png">

Reply:
Needs some spacing after the Google button, and an "or" after the button.

Reply:
Does Remember me work for Google login as well?

Reply:
Nope. I think it will remember you as long as your google account is still signed in to the same browser you're using